src/Command/CommandBase.php

```diff
@@ -1055,6 +1066,72 @@ protected function deleteSshKeyFromCloud(\stdClass $cloud_key): void {
     }
   }
 
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   */
+  protected function checkForNewVersion(InputInterface $input, OutputInterface $output): void {
+    try {
+      if (strpos($input->getArgument('command'), 'api:') === FALSE && $this->hasUpdate()) {
+        $output->writeln("A newer version of Acquia CLI is available. Run <comment>acli self-update</comment> to update.");
```

Comment:
We should use the bold style rather than comments to show the example command. That’s the precedent that we have set elsewhere.